src/main/java/seedu/address/logic/parser/EditGroupCommandParser.java

```diff
@@ -37,7 +37,10 @@ public EditGroupCommand parse(String args) throws ParseException {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
                     EditGroupCommand.MESSAGE_USAGE), ive);
         }
-        String newGroupName = argMultimap.getValue(PREFIX_NAME).orElse("");
+        String newGroupName = "";
+        if (argMultimap.getValue(PREFIX_NAME).isPresent()) {
+            newGroupName = ParserUtil.parseGroupName(argMultimap.getValue(PREFIX_NAME).get());
+        }
```

Comment:
If the user supplies no group name (i.e. they omit the g/ parameter), would this cause the group to have an empty name? I think it would, and that is a problem. We should follow what Person does with the EditPersonDescriptor